install/streams.csv

```diff
@@ -1654,10 +1654,11 @@ QRZ,10,50,false,false,true,1990-04-26T04:39:00Z,2002-02-18T05:58:00Z
 QRZ,10,100,false,true,false,2003-08-06T00:00:00Z,2009-08-12T23:00:00Z
 QRZ,10,100,false,false,false,2009-08-12T23:01:01Z,9999-01-01T00:00:00Z
 QRZ,20,1,false,false,false,2003-08-06T00:00:00Z,2015-11-12T01:00:00Z
-QRZ,20,50,false,false,false,2009-08-12T23:01:01Z,2019-11-21T22:55:00Z
-QRZ,20,200,false,false,true,2003-08-06T00:00:00Z,2019-11-21T22:55:00Z
-QRZ,21,50,false,false,false,2019-11-21T22:56:00Z,2020-06-16T00:00:00Z
-QRZ,21,200,false,false,true,2019-11-21T22:56:00Z,9999-01-01T00:00:00Z
+QRZ,20,50,false,false,false,2009-08-12T23:01:01Z,2015-11-12T01:00:00Z
+QRZ,20,200,false,false,true,2003-08-06T00:00:00Z,2015-11-12T01:00:00Z
+QRZ,21,50,false,false,false,2015-11-12T02:00:00Z,2020-06-16T00:00:00Z
+QRZ,21,200,false,false,true,2015-11-12T01:00:00Z,2020-02-08T23:59:59Z
+QRZ,21,200,false,false,false,2020-02-09T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
- For the LC 21 200hz continuous, where is the start time based on? `2020-02-09T00:00:00Z`

Reply:
@mnaguit from the data daily  list of the HN 200Hz - 2020.040 (09/02)   is when a large bunch of 200Hz  streaming was set  :)